src/ucp/core/ucp_ep.c

```diff
@@ -292,26 +300,21 @@ ucs_status_t ucp_ep_init_create_wireup(ucp_ep_h ep,
     ucp_ep_config_key_reset(&key);
     ucp_ep_config_key_set_params(&key, params);
 
-    /* all operations will use the first lane, which is a stub endpoint */
     key.num_lanes             = 1;
-    key.lanes[0].rsc_index    = UCP_NULL_RESOURCE;
-    key.lanes[0].dst_md_index = UCP_NULL_RESOURCE;
-    key.am_lane               = 0;
-    key.wireup_lane           = 0;
-    key.tag_lane              = UCP_NULL_LANE; /* should be initialized only if
-                                                  HW TM is supported */
-    key.am_bw_lanes[0]        = 0;
-    key.rma_lanes[0]          = 0;
-    key.rma_bw_lanes[0]       = 0;
-    key.amo_lanes[0]          = 0;
-
+    if (ucp_worker_sockaddr_is_cm_proto(ep->worker)) {
+        key.cm_lane           = 0;
+    } else {
+        /* all operations will use the first lane, which is a stub endpoint */
+        key.wireup_lane       = 0;
+        key.am_lane           = 0;
+    }
     status = ucp_worker_get_ep_config(ep->worker, &key, 0, &ep->cfg_index);
     if (status != UCS_OK) {
         return status;
     }
 
-    ep->am_lane               = 0;
-    ep->flags                |= UCP_EP_FLAG_CONNECT_REQ_QUEUED;
+    ep->am_lane  = key.wireup_lane;
```

Comment:
minor: extra space before `=`

Reply:
it's aligned with |= in the next line

Reply:
yes, but the line below also has extra spaces